elide-async/src/main/java/com/yahoo/elide/async/service/thread/TableExportThread.java

```diff
@@ -78,25 +118,39 @@ protected TableExport storeResults(Observable<String> result) {
      * @return result as Observable of String
      */
     protected Observable<String> processObservablePersistentResource(Observable<PersistentResource> resources) {
-        Observable<String> results = Observable.just("No Records Generated");
+        Observable<String> results = Observable.empty();
```

Comment:
We should generate the CSV header using the same code regardless of what the record count is.